### PR TITLE
auto open camera when --input 0

### PR DESCRIPTION
**Pull request by lgynju**

### CODE REVIEW

File: yolo.py [hunk 1]
```diff
@@ -171,6 +171,8 @@ def close_session(self):
 
 def detect_video(yolo, video_path, output_path=""):
     import cv2
+    if (video_path == "0"):
```
Comment (Borda) [inline yolo.py]: in case you have multiple cameras..
```
try:
    video_path = int(video_path)
except Exception:
    pass  # not usiong web cam

Comment (lgynju) [reply yolo.py]: Done

**Comment by Borda** [comment]
good job, it seems that the CodeFactor does not like the `pass` then I would recommend using `video_path = update_path(video_path)` where the function is `yolo3.utils.update_path` :]

**Comment by Borda** [comment]
do not change `yolo3.utils.update_path` just update the detection script
```
try:
    path_video = int(path_video)
except Exception:  # not using web cam
    path_video = update_path(path_video)
```

**Comment by Borda** [comment]
or allow changes in your PR - [Allowing changes to a pull request branch created from a fork](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork)

**Pull request merged**